Tests/TokenizersTests/TokenizerTests.swift

```diff
@@ -95,6 +95,17 @@ class GemmaUnicodeTests: XCTestCase {
     }
 }
 
+class PhiSimpleTests: XCTestCase {
```

Comment:
This test passes but [these ones](https://github.com/huggingface/swift-transformers/blob/43ae46d22a8870d8034129972d4f19dcdbf5ead9/Tests/PreTokenizerTests/PreTokenizerTests.swift#L139) do not. I don't know if these tests were wrong all along or there's something else going on.